src/ReadVTK.jl

```diff
@@ -406,6 +411,23 @@ function get_data_section(vtk_file::VTKFile, section)
   return VTKData(names, data_arrays, vtk_file)
 end
 
+# Retrieve FieldData section from the VTK file
+function get_field(vtk_file::VTKFile)
```

Comment:
```suggestion
function get_field_data(vtk_file::VTKFile)
```
for consistency with `get_cell_data` and `get_point_data`?